modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -68,6 +68,10 @@ def authn_context
       'authn'
     end
 
+    def set_edipi(edipi_header)
```

Comment:
This isn't being called anywhere, why are we adding this method back?

Reply:
could be a merge issue. I will check it out

Reply:
yea, it was added back in the merge. I removed it as well as another redundant method. I also added back the loa assignment that got lost in the shuffle as well.